### ISSUE TITLE
~ accessibility follow UI

### ISSUE THREAD

**Issue by hoodieak** [unascribed/sleeping.town#44]
![image](https://user-images.githubusercontent.com/23704040/44186369-17ae7980-a0e7-11e8-8273-cb4b451eda5f.png)

essentially, the way that the profile image for sleepingstyle means often dont get good colors for accessibility to see. a simple darkish gray ~ black translucent background would make it much much easier to use for my weak eyes.

thanks for listening. (had to include full profile for adorable avatar!!)

**Comment by unascribed** [comment]
yeeah i noticed this when i changed my header to something light

Material Design text protection scrims are kinda bad for accessibility but i did this before i really had realized that

**Comment by hoodieak** [comment]
rooting for yu... might see what ic an do but not got lots of HTML experience

**Comment by unascribed** [comment]
Best I can tell, Sleeping v2 does not exhibit this problem due to increased contrast on decorative labels like this one. On a solid white background they remain fairly legible.
